ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerImpl.java

```diff
@@ -633,6 +633,13 @@ void assertGroup(Object requestorId, RaftGroupId requestorGroupId) throws GroupM
     return pending.getFuture();
   }
 
+  public void stepDownOnJvmPause() {
+    if (isLeader()) {
+      final LeaderState leaderState = role.getLeaderStateNonNull();
+      leaderState.submitStepDownEvent(LeaderState.StepDownReason.JVM_PAUSE);
+    }
```

Comment:
It should use `getLeaderState()` instead of `getLeaderStateNonNull()`
```suggestion
    role.getLeaderState().ifPresent(leader -> leader.submitStepDownEvent(LeaderState.StepDownReason.JVM_PAUSE));
```